pom.xml

```diff
@@ -48,6 +48,7 @@
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 
+		<jbtis.version>4.5.0.AM1-SNAPSHOT</jbtis.version>
```

Comment:
I am talking about that addition of the jbtis.version property. This property doesn't seem to be used anywhere or am I wrong?